database/schema/upgrade_scripts/104-notificator.sql

```diff
@@ -0,0 +1,35 @@
+-- Notifications integration with vulnerability
+CREATE USER ve_db_user_notificator;
+ALTER USER ve_db_user_notificator WITH PASSWORD 've_db_user_notificator_pwd';
+
+CREATE TYPE notification AS enum('new-cve-cvss', 'new-cve-severity', 'new-cve-security-rule', 'any-cve-known-exploit');
+
+CREATE TABLE IF NOT EXISTS notified_accounts (
+  rh_account_id INT NOT NULL,
+  cve_id INT NOT NULL,
+  notif_type notification NOT NULL,
+  UNIQUE (rh_account_id, cve_id, notif_type),
```

Comment:
Make sure the index created by UNIQUE is used also when selecting only by `rh_account_id, cve_id` pair. If not, create additional index on these 2 columns.

Reply:
```
vulnerability=# select * from notified_accounts ;
 rh_account_id | cve_id |      notif_type       
---------------+--------+-----------------------
             1 |      1 | new-cve-cvss
             1 |      2 | new-cve-cvss
             1 |      3 | new-cve-cvss
             1 |      4 | new-cve-cvss
             1 |      5 | new-cve-cvss
             1 |      5 | new-cve-severity
             1 |      4 | new-cve-severity
             1 |      3 | new-cve-severity
             1 |      2 | new-cve-severity
             1 |      1 | new-cve-severity
             1 |      0 | new-cve-severity
             1 |      0 | new-cve-security-rule
             1 |      1 | new-cve-security-rule
             1 |      2 | new-cve-security-rule
             1 |      3 | new-cve-security-rule
             1 |      4 | new-cve-security-rule
             1 |      5 | new-cve-security-rule
(17 rows)

vulnerability=# explain analyze select * from notified_accounts where rh_account_id = 1 and cve_id = 1;
                                                                                  QUERY PLAN                                                                                   
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 Index Only Scan using notified_accounts_rh_account_id_cve_id_notif_type_key on notified_accounts  (cost=0.15..8.17 rows=1 width=12) (actual time=0.042..0.124 rows=3 loops=1)
   Index Cond: ((rh_account_id = 1) AND (cve_id = 1))
   Heap Fetches: 3
 Planning Time: 0.124 ms
 Execution Time: 0.288 ms
(5 rows)

```